.luacheckrc

```diff
@@ -0,0 +1,14 @@
+unused_args = false
+allow_defined_top = true
```

Comment:
Not needed. (really it should never be used, all globals should be explicitly defined)
```suggestion
```

Reply:
Second this, allow_defined_top should never be enabled for mods and not really many legit use cases anywhere.
This test failing (unless it is just some missing known globals) can indicate serious issues with global namespace pollution and should fail (and even Minetest itself will complain for this).